### ISSUE TITLE
WYMEditor completely broken in IE9

### ISSUE THREAD

**Issue by iebuggy** [wymeditor/wymeditor#245]
WYMEditor, including latest git master repo, is completely broken under IE9.  I built from the git repo in the hopes that the issues were fixed.  The demos that come with WYMEditor show the problems rather well.  Here's a rough overview of the big breakages that make this a deal-breaker:

1)  Caret doesn't move between characters.  It is either at the start of a line or the end of a line.
2)  Pressing enter inserts weird paragraph tags.
3)  Clicking a 'p' tag causes a weird selection box to show up.  Double-clicking it enters some bizarre editor mode.
4)  Scrollbars never vanish.  Inner content area is always wider than the iframe.

Basically, it is completely unusable.
